add-roommate-searcher-on-board

amit-schwartz-u opened the pull request:
PR changes:
1. opened Roommate searcher user flow.
2. changed set profile to set basic apartment info (still without other roommates info and without filters)
3. changed apartment entryDate from int to string

amit-schwartz-u committed:
<!-- please don't check these files.-->

amit-schwartz-u committed:
Merge branch 'master' of https://github.com/amit-schwartz-u/rooMe

 Conflicts:
	app/src/main/java/com/example/roome/MainActivityRoommateSearcher.java
	app/src/main/java/com/example/roome/Roomate_searcher_tabs_classes/EditFiltersRoommateSearcher.java
	app/src/main/java/com/example/roome/Roomate_searcher_tabs_classes/EditProfileRoommateSearcher.java
	app/src/main/java/com/example/roome/Roomate_searcher_tabs_classes/MatchesRoommateSearcher.java
	app/src/main/java/com/example/roome/Roomate_searcher_tabs_classes/RoommateSearcherHome.java
	app/src/main/java/com/example/roome/RoommateSearcherSetProfileActivity.java

amit-schwartz-u committed:
Merge branch 'master' of https://github.com/amit-schwartz-u/rooMe

amit-schwartz-u committed:
changes:
1. opened Roommate searcher user flow.
2. changed set profile to set basic apartment info (still without other roommates info and without filters)
3. changed apartment entryDate from int to string

amit-schwartz-u committed:
remove irrelevant comments

amit-schwartz-u committed:
remove irrelevant comment

ShaharBirzon commented on app/src/main/java/com/example/roome/ChoosingActivity.java:
```diff
@@ -136,16 +136,16 @@ private void updateUserName() {
+        String key = newRef.getKey();
+        newRef.setValue(userObj);
+        MyPreferences.setUserUid(getApplicationContext(), key);
+        while (!done.get()) ;
```
הלולאת הזו קצת מוזרה ויש לי תחושה שאפשר לפתור את זה יותר יפה.. אולי אפשר ליצור מחלקה שיורשת מאפליקיישן כמו שלמדנו בפוסט פי סי ששם יהיו כל האיתחולים והבדיקות שכל המידע נמצא...

ShaharBirzon commented on app/src/main/java/com/example/roome/RoommateSearcherSetProfileActivity.java:
```diff
@@ -1,18 +1,25 @@
 package com.example.roome;
```
why you didn't do it in the class EditProfileRoommateSearcher that is already created?

amit-schwartz-u replied:
Because it's not the same thing. you are right that I need to change this name of activity because we decided to make it at an apartment setup. 

I'll rename the activity to roommateSearcherSetApartmentInfo

ShaharBirzon commented on app/src/main/java/com/example/roome/RoommateSearcherSetProfileActivity.java:
```diff
@@ -75,16 +97,19 @@ protected void onCreate(Bundle savedInstanceState) {
-                uploadPhotoOnClickAS();
-            }
-        });
+
```
אפשר למחוק את זה או שצריך לאחכ?

amit-schwartz-u replied:
we need it in a different place in this class. I'll remove it for now :)

ShaharBirzon commented on app/src/main/res/layout/activity_roommate_searcher_set_profile.xml:
```diff
@@ -1,64 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- please don't check this class.-->
```
אפשר להעיף את ההערה הזו

ShaharBirzon commented on app/src/main/res/layout/activity_roommate_searcher_set_profile.xml:
```diff
@@ -68,6 +45,218 @@
+                app:layout_constraintStart_toStartOf="@+id/cl_details"
+                app:layout_constraintTop_toBottomOf="@+id/tv_apartment_neighborhood" />
+
+            <ImageView
+                android:id="@+id/iv_sep_line1"
+                android:layout_width="match_parent"
+                android:layout_height="2dp"
+                android:layout_marginStart="15dp"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="15dp"
+                app:srcCompat="@drawable/ic_seperating_line"
+                app:layout_constraintBottom_toTopOf="@+id/tv_apartment_entry_date"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/spinner_neighborhood" />
```
תחליף את זה ל:
 <View
        android:id="@+id/sep_line1"
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_marginStart="15dp"
        android:layout_marginEnd="15dp"
        android:layout_marginTop="16dp"
        android:background="@android:color/black"
        app:layout_constraintBottom_toTopOf="@+id/tv_apartment_entry_date"
        app:layout_constraintEnd_toEndOf="parent
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toBottomOf="@id/spinner_neighborhood" />

ShaharBirzon commented on app/src/main/res/layout/activity_roommate_searcher_set_profile.xml:
```diff
@@ -68,6 +45,218 @@
+                app:layout_constraintStart_toStartOf="@+id/iv_choose_apartment_entry_date"
+                app:layout_constraintTop_toTopOf="@+id/iv_choose_apartment_entry_date" />
+
+            <ImageView
+                android:id="@+id/iv_sep_line2"
+                android:layout_width="match_parent"
+                android:layout_height="2dp"
+                android:layout_marginStart="15dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginEnd="15dp"
+                app:srcCompat="@drawable/ic_seperating_line"
+                app:layout_constraintBottom_toTopOf="@+id/tv_choose_num_of_roommates"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/iv_choose_apartment_entry_date" />
```
same

ShaharBirzon commented on app/src/main/res/layout/activity_roommate_searcher_set_profile.xml:
```diff
@@ -68,6 +45,218 @@
+                    android:textSize="20sp" />
+            </RadioGroup>
+
+            <ImageView
+                android:id="@+id/iv_sep_line3"
+                android:layout_width="match_parent"
+                android:layout_height="2dp"
+                android:layout_marginStart="15dp"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="15dp"
+                app:srcCompat="@drawable/ic_seperating_line"
+                app:layout_constraintBottom_toTopOf="@+id/tv__apartment_rent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/radioGroup_num_of_roommates" />
```
same

amit-schwartz-u committed:
changes:
1. fixed (renamed) roommate tabs packge.
2. changed the roommateSearcherUser onBoarding to editProfile.
